manual/table/content_and_subtables.rb

```diff
@@ -19,17 +19,16 @@
 #
 require File.expand_path(File.join(File.dirname(__FILE__),
                                    %w[.. example_helper]))
-
 filename = File.basename(__FILE__).gsub('.rb', '.pdf')
 Prawn::ManualBuilder::Example.generate(filename) do
+  image_path = File.expand_path(File.join(File.dirname(__FILE__), %w[data images stef.jpg]))
```

Comment:
You can use `__dir__` constant.

Reply:
For some reason `__dir__` inside the manual examples is being evaluated to `nil`, so I've use it in the specs, but not here.